draft-ietf-ppm-dap.md

```diff
@@ -1758,9 +1767,28 @@ The Leader and Helper initialization behavior is detailed below.
 
 #### Leader Initialization {#leader-init}
 
+~~~ aasvg
+                     PrepareResp
+Report --> Continued -----+----> Continued
+                          |
+                          +----> FinishedWithOutbound
+                          |
+                          +----> Finished (*commit)
+                          |
+                          +----> Rejected (*reject)
+~~~
+{: #leader-init-state title="Leader state transition triggered by aggregation initialization. (\*) indicates a terminal state." }
+
 The Leader begins an aggregation job by choosing a set of candidate reports that
 belong to the same task and a job ID which MUST be unique within the task.
 
+First, the Leader MUST ensure each report in the candidate set can be
+committed. That is: the Leader MUST ensure it has not committed a report with
+the same ID and task; and the Leader MUST ensure the batch bucket to which the
+report belongs has not been collected. See {{batch-buckets}} for details. If a
+report is not cannot be committed, then the Leader rejects it and removes it
+from the candidate set.
```

Comment:
Leader MUST ensure commitment won't fail. This amounts to resolving race conditions across aggregation jobs and collection jobs, both of which are already operational requirements.